packages/shadydom/src/patch-native.js

```diff
@@ -110,6 +110,13 @@ export const addNativePrefixedProperties = () => {
   ];
   if (window.EventTarget) {
     copyProperties(window.EventTarget.prototype, eventProps);
+
+    // In Firefox 33, `EventTarget` exists, but `EventTarget.prototype` is not
+    // in the prototype chain of `window` and, strangely,
+    // `window instanceof EventTarget` still returns true.
+    if (window[NATIVE_PREFIX + 'addEventListener'] === undefined) {
```

Comment:
Does `EventTarget` exist in `Node.prototype`? (the else clause also deals with patching Node up)

Reply:
Yes, `Node` seems to be set up correctly:

![Screen Shot 2021-01-13 at 14 44 33](https://user-images.githubusercontent.com/406614/104519304-1599bf00-55ae-11eb-839b-70ed1e720198.png)

Reply:
Ok, thanks for checking.